bee-pow/README.md

```diff
@@ -0,0 +1 @@
+# bee-pow
```

Comment:
I agree – it's something I've tried to do with the FPC stuff, and I think it would be useful to have everywhere. Some summary comments and perhaps an example of how to use the crate functionality would go a log way.

Reply:
I won't have time to do that and we need ti merged. I added a simple sentence for now and we'll improve later.

Reply:
okay :+1: